cf-agent/verify_files_utils.c

```diff
@@ -3596,13 +3596,13 @@ bool VerifyOwner(EvalContext *ctx, const char *file, const Promise *pp, Attribut
 
     if (attr.perms.groups->next == NULL && attr.perms.groups->gid == CF_UNKNOWN_GROUP)  // Only one non.existent item
     {
-        cfPS(ctx, LOG_LEVEL_ERR, PROMISE_RESULT_FAIL, pp, attr, "Unable to make file belong to an unknown group");
+        cfPS(ctx, LOG_LEVEL_ERR, PROMISE_RESULT_FAIL, pp, attr, "Unable to make file belong to an unknown group '%s'", attr.perms.groups->gid );
         *result = PromiseResultUpdate(*result, PROMISE_RESULT_FAIL);
```

Comment:
The reason for having `CF_UNKNOWN_OWNER` in the first place, is that the username/groupname the promise asks for doesn't exist. Maybe the message should be better.